index.html

```diff
@@ -161,6 +161,41 @@ <h1>The Registration Process</h1>
 the modification request adheres to the following rules:
     </p>
 
+    <p>
+      The JSON representation of a DID Document is the same document (byte for byte equivalent) 
+      as the JSON-LD representation of a DID Document. 
+      JSON representations rely on the JSON-LD vocabulary (contained in this registry) for interoperability. 
+    </p>
+
+    <p>
+      Due to the nature of decentralization, JSON documents that contain terms 
+      not defined in a JSON-LD impossible to prevent.
+      Managing the validity / security considerations associated with unknown / undefined 
```

Comment:
This is tied to the "what do we do with unknown properties" discussion currently going on, we don't have finality on that yet.

Reply:
agreed... however, i believe it remains out of our control... we cannot prevent unknown properties... only the software that constructs did documents can.

Reply:
However we could mandate that both JSON and JSON-LD ignore unrecognized members and write test cases for that by including random members and making sure that errors aren't thrown.  But then, this isn't really the topic of this PR...

Reply:
I believe the new language addresses is much more clearly, resolving, but feel free to start a new thread on the revised language.